src/util/StringUtils.cpp

```diff
@@ -5,12 +5,8 @@
 #include <string.h>
 
 string StringUtils::toLowerCase(string input)
-{
-	char* lower = g_utf8_strdown(input.c_str(), input.size());
-	string strLower = lower;
-	g_free(lower);
-
-	return strLower;
+{	
```

Comment:
This produces a memory leak

https://developer.gnome.org/glib/stable/glib-Unicode-Manipulation.html#g-utf8-strdown

Reply:
Please be really carefull with Memory stuff, such bugs are really hard to find

Reply:
Ok, so it needs to be like before this change?

Reply:
Yes.